### ISSUE TITLE
Feature request: Control of y-axis

### ISSUE THREAD

**Issue by sjspielman** [cjbattey/driftR#1]
Hi @cjbattey, 

First, thanks for a great package!! I'm wondering if it would be possible to implement a feature where users can optionally control the y-axis limits, or instead change the default behavior to go from 0-1 for all simulation results.
For example, attached simulations of 1) balancing selection <img width="863" alt="screen shot 2018-02-05 at 2 19 37 pm 2" src="https://user-images.githubusercontent.com/4701111/35824129-d5927fa4-0a7f-11e8-953e-164f19c38542.png">, and 2) low-frequency <img width="858" alt="screen shot 2018-02-05 at 2 20 34 pm 2" src="https://user-images.githubusercontent.com/4701111/35824137-def66ac4-0a7f-11e8-8e00-b25c9b79e617.png"> alleles appear misleading. In the first case, allele frequencies appear to be jumping all around but are really only in a small y-axis range about 0.5. In the second, it appears that an allele may have gone to fixation, but really it is still of relatively low frequency in the population.

Thanks for your consideration!
-Stephanie

**Comment by cjbattey** [comment]
Hi Stephanie, 

Thanks for the suggestion. I'll add a button to freeze the y axes to (0,1) - should be up by this afternoon. Best, 

CJ

**Comment by sjspielman** [comment]
Thanks for your quick response!

**Comment by sjspielman** [comment]
Also, while I'm here, I wanted to ask you - is there a particular reason why 2 populations are required for a given simulation, as opposed to 1? The simulator errors out if at least 2 aren't specified.

I'm happy to create a separate Issue if you think this comment would merit one. Thanks!

**Comment by cjbattey** [comment]
Hi Stephanie, 

y-axis control is now in the latest commit and available on the live shiny site, and I set the default to keep all axes at (0,1). 

I also fixed the bug that was causing single-population runs to fail - it was rowMeans() expecting a data.frame when calculating allele frequencies. Thanks for pointing these out - don't hesitate to open another issue for other feature requests/bug fixes in the future! 

CJ
